src/ex4/dist/app.js

```diff
@@ -0,0 +1,102 @@
+const addButton = document.getElementById("list-item-submit")
+const textInput = document.getElementById("list-item-input")
+const allLists = document.getElementById("list")
+const counterLists = document.getElementById("counter")
+const errorMessage = document.getElementById("error")
+const regex = /^[0-9]*$/
```

Comment:
regex is not a descriptive name - I suppose you use it for pokemon ID identification